rakelib/fix_nod_evidence_submissions.rake

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'decision_review_v1/service'
+
+namespace :decision_reviews do
+  desc 'Enqueue jobs for NOD V2 submissions with evidence uploads that did not get uploaded to Lighthouse'
+  task fix_nod_evidence_submissions: :environment do
+    # The bulk of the code is below (line 55) and the UserObject class and complete_nod_evidence_submissions method
+    # are helpers
+
+    # This is a proxy for the User object that we do not have access to in this rake task
+    class UserObject
```

Comment:
I'm getting a `W: Lint/ConstantDefinitionInBlock: Do not define constants this way within a block.` on this from Rubocop — any suggestions for fixing that?

cc: @data-doge